### PR TITLE
fix(ActivityCenter): Fix CommunityBadge eliding

### LINKED ISSUE #9402: Limit notification badge size
Screenshot 2023-02-02 at 13 24 47

### PR DESCRIPTION
**Pull request by MishkaRogachev**
Close #9402

### What does the PR do

Make CommunityBadge respect long community and channel names

### Affected areas

ActivityCenter

### Screenshot of functionality (including design for comparison)

Screenshot 2023-02-15 at 16 22 27

### CODE REVIEW

**Review by caybro — APPROVED** [review]
LGTM

**Comment by MishkaRogachev** [comment]
@caybro @osmaczko re-requesting review because of new component

File: ui/StatusQ/src/StatusQ/Controls/StatusLinkText.qml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    /*!
+       \qmlproperty StatusLinkText StatusLinkText::containsMouse
+       This property true whenever text is hoverd by mouse cursor
+    */
+    readonly property alias containsMouse: textMouseArea.containsMouse
+
+    signal clicked()
+
+    maximumLineCount: 1
+    wrapMode: Text.Wrap
```
Comment (caybro) [inline ui/StatusQ/src/StatusQ/Controls/StatusLinkText.qml:39]: `wrapMode` doesn't make much sense with `maximumLineCount: 1` :)

File: ui/app/mainui/activitycenter/controls/CommunityBadge.qml [hunk 2]
```diff
@@ -86,20 +81,12 @@ Badge {
...
-                readOnly: true
-                textFormat: Text.RichText
-                clip: true
-                color: Theme.palette.baseColor1
-                font.pixelSize: 13
-                font.weight: Font.Medium
-                onLinkActivated: {
-                    root.channelNameClicked()
-                }
+                text: "#" + root.channelName
```
Comment (caybro) [inline ui/app/mainui/activitycenter/controls/CommunityBadge.qml:88]: Where has `Utils.getLinkStyle()` disappeared?

File: ui/app/mainui/activitycenter/controls/CommunityBadge.qml [hunk 3]
```diff
@@ -61,21 +63,14 @@ Badge {
...
-                readOnly: true
-                textFormat: Text.RichText
-                clip: true
-                color: Theme.palette.baseColor1
-                font.pixelSize: 13
-                font.weight: Font.Medium
-                onLinkActivated: {
-                    root.communityNameClicked()
-                }
+                text: root.communityName
```
Comment (caybro) [inline ui/app/mainui/activitycenter/controls/CommunityBadge.qml:72]: Same here, no more style?

**Review by caybro — CHANGES_REQUESTED** [review]

File: ui/app/mainui/activitycenter/controls/CommunityBadge.qml (hunk 2 above)
Comment (MishkaRogachev) [reply ui/app/mainui/activitycenter/controls/CommunityBadge.qml:88]: Yes, now it's useless :)

File: ui/StatusQ/src/StatusQ/Controls/StatusLinkText.qml (hunk 1 above)
Comment (MishkaRogachev) [reply ui/StatusQ/src/StatusQ/Controls/StatusLinkText.qml]: fixed

File: ui/app/mainui/activitycenter/controls/CommunityBadge.qml (hunk 2 above)
Comment (MishkaRogachev) [reply ui/app/mainui/activitycenter/controls/CommunityBadge.qml:88]: https://user-images.githubusercontent.com/2522130/219346744-faf93b1d-5764-4aa0-b79b-310d1a2e92b2.mov

Comment (caybro) [reply ui/app/mainui/activitycenter/controls/CommunityBadge.qml:88]: The link should be blue when hovered, it's not useless 😊

Comment (MishkaRogachev) [reply ui/app/mainui/activitycenter/controls/CommunityBadge.qml:88]: image
It was overriden, but i can add blue on hover in StatusQ component

Comment (caybro) [reply ui/app/mainui/activitycenter/controls/CommunityBadge.qml:88]: Yeah, you can add a `linkColor` value to `StatusLinkText`

Comment (MishkaRogachev) [reply ui/app/mainui/activitycenter/controls/CommunityBadge.qml:88]: Screenshot 2023-02-17 at 12 31 08
@caybro fixed

**Review by osmaczko — APPROVED** [review]
LGTM

File: ui/app/mainui/activitycenter/controls/CommunityBadge.qml (hunk 2 above)
Comment (caybro) [reply ui/app/mainui/activitycenter/controls/CommunityBadge.qml:88]: Cool, thx!

**Review by caybro — APPROVED** [review]
LGTM

**Pull request merged**